config/spring19_seminar/bert.conf

```diff
@@ -0,0 +1,40 @@
+// Import the defaults using the relative path
+include "../final.conf"
+
+
+// Output path
+project_dir = ${JIANT_PROJECT_PREFIX}
+
+
+// Optimization
+batch_size = 16
+dropout = 0.1 // following BERT paper
+lr = 2e-5  // following Jason, Alex
+max_epochs = 3 // If positive, maximum number of epochs
+val_interval = 100
+eval_val_interval = 100
+
+
+// Target tasks
+do_target_task_training = 1  // If true, after do_pretrain train the task-specific model parameters
+write_preds = "val,test"  // 0 for none, or comma-separated splits in {"train", "val", "test"} 
+                          // for which predictions are written to disk during do_full_eval
+
+
+// Pretraining tasks
+load_model = 0  // If true, restore from checkpoint when starting do_pretrain
+
+
+// Models
+// Model, BERT
+tokenizer = "bert-large-cased"
+sent_enc = "null" // "bow", "rnn" for LSTM, "null"
```

Comment:
Maybe change to sent_enc = "none" and also clean up the comments?

Reply:
seems that `sentence_encoder` function in `models.py` makes references to `null`

Reply:
The change from 'null' to 'none' was part of a recent bug fix—are you sure you're using the latest version of master? I don't see 'null' (in quotes) anywhere in models.py.

Reply:
will fix this

Reply:
done!